src/gluonts/dataset/common.py

```diff
@@ -31,6 +31,7 @@
 import pandas as pd
 import pydantic
 from pandas.tseries.offsets import Tick
+from typing_extensions import Protocol
```

Comment:
I wonder if we need a `requirement` for this.

This seems to be pulle in via either `black` or `mypy` which are not actual requirements of gluonts.

Reply:
Mh good point, didn't notice...